Application/Frontend/components/FriendsColumn/FriendsTab.tsx

```diff
@@ -111,48 +110,53 @@ export function FriendsTab(props: TextInputProps) {
         };
         fetchData(); 
       }, []);
+
+          // Filter friendList based on search query
+    const filteredFriendList = friendList.filter((friend) =>
```

Comment:
I really like this real time filtering feature you have implemented—it really benefits the user experience! Awesome work!